bundles/net.tourbook/src/net/tourbook/ui/views/tourDataEditor/TourDataEditorView.java

```diff
@@ -6228,6 +6367,7 @@ private void enableActions() {
                   && (_isManualTour == false));
       _actionToggleReadEditMode.setEnabled(isCellEditorInactive && isTourInDb);
 
+      final boolean canEdit = _isEditMode && isTourInDb;
```

Comment:
I think Eclipse did that. Eclipse sometimes changes lines without notice. I spent 30+ mins reverting back `TourData.java`, `TourDataEditorView.java`, `TourDatabase.java` because a lot of lines were misplaced. I still don't understand how to prevent Eclipse from doing that and it is annoying. (I don't have this problem in any other Dev tools like Visual Studio etc...)

Reply:
Fixed

Reply:
When fields are reformatted in Eclipse, only this helps:
* Add // comments for empty lines
* Format manually with the corresponding tags

When a method is commented out, then Eclipse is sometimes moving this comment to another method position, so I try to not comment it out and add a _UPPERCASE_POSTFIX_TAG to keep it

Reply:
> I think Eclipse did that. Eclipse sometimes changes lines without notice. I wasted 30+ mins reverting back `TourData.java`, `TourDataEditorView.java`, `TourDatabase.java` because a lot of lines were misplaced. I still don't understand how to prevent Eclipse from doing that and it is annoying. (I don't have this problem in any other Dev tools like Visual Studio etc...)

As I already said, since a while I review all code before I do a commit, this takes some time but it prevents wrong formatting and it also helps to remove temporarily comments